src/System/Wlog/Wrapper.hs

```diff
@@ -40,10 +40,11 @@ import           System.Wlog.Severity       (Severity (..))
 streamHandlerWithLock :: (Handle -> Text -> IO ())
                       -> MVar ()
                       -> Handle
+                      -> (Handle -> Bool)
                       -> Severity
                       -> IO (GenericHandler Handle)
-streamHandlerWithLock customTerminalAction lock handle severity =
-    streamHandler handle customTerminalAction lock severity
+streamHandlerWithLock customTerminalAction lock handle shouldPrintError severity =
```

Comment:
`streamHandlerWithLock` function is not needed because all it does is just calls `streamHandler` with the same arguments. This function is left after my refactoring... It did more previously but now there's no actual need in this function as I see.

Reply:
Great! I also don't see any sense to have this one.